persim/persistent_entropy.py

```diff
@@ -0,0 +1,61 @@
+"""
+
+    Implementation of persistent entropy
+
+    Author: Eduardo Paluzo Hidalgo (cimagroup, University of Seville)
+    contact: epaluzo@us.es
+
+"""
+
+import numpy as np
+
+
+__all__ = ["persistent_entropy"]
+
+def pentropy(dgms, inf = 0, valInf = -1, norm = 0):
```

Comment:
Use boolean explicitly for `norm`, and please change to a verb. I think the full word `normalize` would make more sense.